Enhancing installation instructions (#9392)
fishtown-analytics/docs.getdbt.com: website/docs/docs/local/about-local.md

---
title: About dbt local installations
id: about-local
description: "dbt tools and resources that can be installed on your local machine"
sidebar_label: "About local installation"
pagination_next: "docs/local/install-dbt"
pagination_prev: null
---

> dbt enables data teams to transform data using analytics engineering best practices. You can run dbt locally through a command line interface (CLI) to build, test, and deploy your data transformations. The following tools are available for local installation on your computer:

Reviewer: Hiya Matt

The intro says you run dbt locally “through a command line interface (CLI)”, but this page covers four tools, not just the CLI. Maybe the intro can be broadened so it matches the full page?

- **Refactoring tools:** Rename models and columns across your project.

This is the fastest way to get started with dbt locally.

[Install Fusion with the dbt VS Code extension](/docs/local/install-dbt)

## dbt Core

dbt Core is the open-source engine for running dbt locally. It is available in two versions:

- [<Constant name="core_v1" />](/docs/local/install-dbt?version=1): The original Python-based dbt engine with a rich set of features.
- [<Constant name="core_v2" />](/docs/dbt-versions/core-upgrade/upgrading-to-v2) <Lifecycle status="alpha" />: The next major version, built on the Fusion runtime. Currently in alpha.

[<Constant name="core_v1" />](/docs/local/install-dbt?version=1) includes:

- **Apache License 2.0:** <Constant name="core" /> is open source now and forever.
- **Community adapters:** An amazing community of contributors has built adapters for a vast [catalog of data warehouses](/docs/supported-data-platforms).
- **Code editor support:** Build your dbt project in popular editors like VS Code or Cursor.
- **Command line interface:** Run your project from the terminal using macOS Terminal, iTerm, or the integrated terminal in your code editor.

> [Install dbt Core now!](/docs/local/install-dbt?version=1#installation)

Reviewer: Hiya Matt, this link points to the vs code extension section

## dbt Wizard

> [dbt Wizard](/docs/dbt-ai/wizard-quickstart) is a natural next step for local dbt development. It works with both <Constant name="fusion_engine" /> and <Constant name="core" /> — [dbt Wizard](/docs/dbt-ai/wizard-quickstart) adds an AI agent that works with a live understanding of your entire project through dbt's [native metadata engine](/docs/dbt-ai/about-dbt-ai) — a structured index of your [lineage](/docs/explore/explore-projects), model health, test coverage, and semantic definitions.

Reviewer: ```suggestion
[dbt Wizard](/docs/dbt-ai/wizard-quickstart) is a natural next step for local dbt development. It works with both <Constant name="fusion_engine" /> and <Constant name="core" />. dbt Wizard adds an AI agent that understands your full project through dbt's [native metadata engine](/docs/dbt-ai/about-dbt-ai), a structured index of your [lineage](/docs/explore/explore-projects), model health, test coverage, and semantic definitions.

```

- **Build and refactor from natural language**: Describe the change, get a reviewable diff, approve before anything is written.
- **Validate in a tight loop**: Every proposed change compiles and runs against your warehouse, catching issues before production.
- **Navigate with full project context**: Traverse the [DAG](/docs/explore/explore-projects), surface downstream impact, and keep tests and YAML in sync as models evolve.

> For data practitioners, <Constant name="wizard"/> adds an AI layer that knows your project — not just your code. See [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to get started.

Reviewer: ```suggestion
For data practitioners, <Constant name="wizard"/> adds an AI layer that knows your project, not just your code. Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to get started.
```

> [dbt Wizard](/docs/dbt-ai/wizard-quickstart) also works with <Constant name="core" />. See the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to add AI-assisted development to your Core workflow.

Reviewer: ```suggestion
[dbt Wizard](/docs/dbt-ai/wizard-quickstart) also works with <Constant name="core" />. Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to add AI-assisted development to your Core workflow.
```

> [dbt Wizard](/docs/dbt-ai/wizard-quickstart) also works with <Constant name="core" />. Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to add AI-assisted development to your Core workflow.

Reviewer: Hiya Matt

The Wizard section already says it works with both Fusion and Core in the opening paragraph, but the closing paragraph repeats that it also works with Core. That feels redundant, Also, there's three quickstart links close together. Maybe cut the closing paragraph / keep just one “get started” link at the end?

> ## dbt MCP Server

Reviewer: ```suggestion
## dbt MCP server
```

---
title: About dbt local installations
id: about-local
description: "dbt tools and resources that can be installed on your local machine"
sidebar_label: "About local installation"
pagination_next: "docs/local/install-dbt"
pagination_prev: null
---

> dbt enables data teams to transform data using analytics engineering best practices. You can run dbt locally through a command line interface (CLI) to build, test, and deploy your data transformations. The following tools are available for local installation on your computer:

Author: ```suggestion
dbt enables data teams to transform data using analytics engineering best practices. You can run dbt locally through a command line interface (CLI) to build, test, and deploy your data transformations, and install additional tools to enhance your workflows. 
```

## dbt Fusion engine

For the best local development experience, we recommend the <Constant name="fusion_engine" />. Built in Rust, <Constant name="fusion" /> delivers:

- **Faster performance:** Up to 10x faster parsing, compilation, and execution.
- **SQL comprehension:** Dialect-aware validation catches errors before they reach your warehouse.
- **Column-level lineage:** Trace data flow across your entire project.

> [Install Fusion now!](/docs/local/install-dbt)

Reviewer: ```suggestion
[Install <Constant name="fusion"/> now!](/docs/local/install-dbt)
```

### dbt VS Code extension

The [dbt VS Code extension](/docs/dbt-extension-features) combines <Constant name="fusion" />'s performance with powerful <Term id="lsp"/> editor features:

- **IntelliSense:** Autocomplete for models, macros, and columns.
- **Inline errors:** See SQL errors as you type.
- **Hover insights:** View model definitions and column info without leaving your code.
- **Refactoring tools:** Rename models and columns across your project.

This is the fastest way to get started with dbt locally.

> [Install Fusion with the dbt VS Code extension](/docs/local/install-dbt)

Reviewer: ```suggestion
[Install <Constant name="fusion" /> with the dbt VS Code extension](/docs/local/install-dbt)
```

## dbt Core

> dbt Core is the open-source engine for running dbt locally. It is available in two versions:

Reviewer: ```suggestion
<Constant name="core" /> is the open-source engine for running dbt locally. It is available in two versions:
```

- [<Constant name="core_v1" />](/docs/local/install-dbt?version=1): The original Python-based dbt engine with a rich set of features.

> - [<Constant name="core_v2" />](/docs/dbt-versions/core-upgrade/upgrading-to-v2) <Lifecycle status="alpha" />: The next major version, built on the Fusion runtime. Currently in alpha.

Reviewer: ```suggestion
- [<Constant name="core_v2" />](/docs/dbt-versions/core-upgrade/upgrading-to-v2) <Lifecycle status="alpha" />: The next major version, built on the <Constant name="fusion"/> runtime. Currently in alpha.
```

[<Constant name="core_v1" />](/docs/local/install-dbt?version=1) includes:

- **Apache License 2.0:** <Constant name="core" /> is open source now and forever.
- **Community adapters:** An amazing community of contributors has built adapters for a vast [catalog of data warehouses](/docs/supported-data-platforms).
- **Code editor support:** Build your dbt project in popular editors like VS Code or Cursor.
- **Command line interface:** Run your project from the terminal using macOS Terminal, iTerm, or the integrated terminal in your code editor.

> [Install dbt Core now!](/docs/local/install-dbt?version=1#installation)

Reviewer: ```suggestion
[Install <Constant name="core" />now!](/docs/local/install-dbt?version=1#installation)
```

> [Install <Constant name="core" />now!](/docs/local/install-dbt?version=1#installation)

Author: ```suggestion
[Install <Constant name="core" />now!](/docs/local/install-dbt)
```

## dbt Wizard

[dbt Wizard](/docs/dbt-ai/wizard-quickstart) is a natural next step for local dbt development. It works with both <Constant name="fusion_engine" /> and <Constant name="core" />. dbt Wizard adds an AI agent that understands your full project through dbt's [native metadata engine](/docs/dbt-ai/about-dbt-ai), a structured index of your [lineage](/docs/explore/explore-projects), model health, test coverage, and semantic definitions.

- **Build and refactor from natural language**: Describe the change, get a reviewable diff, approve before anything is written.
- **Validate in a tight loop**: Every proposed change compiles and runs against your warehouse, catching issues before production.
- **Navigate with full project context**: Traverse the [DAG](/docs/explore/explore-projects), surface downstream impact, and keep tests and YAML in sync as models evolve.

For data practitioners, <Constant name="wizard"/> adds an AI layer that knows your project, not just your code. Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to get started.

> [dbt Wizard](/docs/dbt-ai/wizard-quickstart) also works with <Constant name="core" />. Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to add AI-assisted development to your Core workflow.

Author: ```suggestion
Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to add AI-assisted development to your Core workflow.
```

> [dbt Wizard](/docs/dbt-ai/wizard-quickstart) also works with <Constant name="core" />. Refer to the [dbt Wizard quickstart](/docs/dbt-ai/wizard-quickstart) to add AI-assisted development to your Core workflow.

Author: ```suggestion
```

## dbt MCP server

The dbt MCP server connects your local dbt project to AI assistants using the [Model Context Protocol](https://modelcontextprotocol.io/). It works with both <Constant name="fusion_engine" /> and <Constant name="core" />, and requires no repository clone.

> - **dbt CLI tools:** Run `dbt run`, `build`, `test`, `compile`, `list`, `parse`, and `show` directly from your AI assistant's chat interface.

Reviewer: ```suggestion
- **<Constant name="platform_cli" /> tools:** Run `dbt run`, `build`, `test`, `compile`, `list`, `parse`, and `show` directly from your AI assistant's chat interface.
```
